src/test/scala/org/ergoplatform/utils/ErgoTestConstants.scala

```diff
@@ -83,7 +83,9 @@ trait ErgoTestConstants extends ScorexLogging {
   val EmptyStateRoot: ADDigest = ADDigest @@ Array.fill(HashLength + 1)(0.toByte)
   val EmptyDigest32: Digest32 = Digest32 @@ Array.fill(HashLength)(0.toByte)
   val defaultDifficultyControl = new LinearDifficultyControl(settings.chainSettings)
-  val defaultExtension: ExtensionCandidate = ExtensionCandidate(Seq(Array(0: Byte, 8: Byte) -> EmptyDigest32))
+  val defaultExtension: ExtensionCandidate = ExtensionCandidate(Seq(
+    Array(0: Byte, 8: Byte) -> EmptyDigest32,
+    Array(1: Byte, 0: Byte) -> Array.fill(HashLength + 1)(0.toByte)))
```

Comment:
Addition of interlink here was required for serialization tests as we can not have an empty `interlinksProof` in the `PoPowHeader`